content/en/monitors/create/types/ci_pipelines.md

```diff
@@ -0,0 +1,119 @@
+---
+title: CI Pipeline Monitor
+kind: documentation
+aliases:
+- /monitors/monitor_types/ci_pipelines
+further_reading:
+- link: "/monitors/notify/"
+  tag: "Documentation"
+  text: "Configure your monitor notifications"
+- link: "/monitors/notify/downtimes/"
+  tag: "Documentation"
+  text: "Schedule a downtime to mute a monitor"
+- link: "/monitors/manage/status/"
+  tag: "Documentation"
+  text: "Check your monitor status"
+---
+
+## Overview
+
+Once [CI Visibility is enabled][1] for your organization, you can create a CI Pipeline monitor to alert you when a specified type of CI Pipeline event exceeds a user-defined threshold over a given period of time.
+
+<div class="alert alert-info"><strong>Note</strong>: Only available in closed beta on US1, EU, US3 and US5.</div>
+
+## Monitor creation
+
+To create a [CI Pipeline monitor][2] in Datadog, use the main navigation: *Monitors -> New Monitor --> CI Pipelines*.
+
+<div class="alert alert-info"><strong>Note</strong>: There is a default limit of 1000 CI Pipeline monitors per account. <a href="/help/">Contact Support</a> to lift this limit for your account.</div>
+
+### Define the search query
+
+1. Construct a search query using the same logic as a CI Pipeline explorer search.
+2. Select the CI Pipeline events level to monitor:
+    * **Monitor over the `Pipeline` level**: If the `Pipeline` level is selected, the monitor will check the events that
+    * **Monitor over the `Stage` level**: If the `Stage` level is selected, the monitor will only evaluate the search query in the stages of every pipeline.
+    * **Monitor over the `Job` level**: If the `Job` level is selected, the monitor will only evaluate the search query in the jobs of every pipeline.
+    * **Monitor over the `Command` level**: If the `Command` level is selected, the monitor will only evaluate the search query in the custom commands executed in every pipeline.
+    * **Monitor over all the levels**: If the `All` level is selected, the monitor will evaluate the search query in all the described levels before.
+2. Choose to monitor over a CI Pipeline event count, facet, or measure:
+    * **Monitor over a CI Pipeline event count**: Use the search bar (optional) and do **not** select a facet or measure. Datadog evaluates the number of CI Pipeline events over a selected time frame, then compares it to the threshold conditions.
+    * **Monitor over a facet**: If a facet is selected, the monitor alerts over the `Unique value count` of the facet.
```

Comment:
what do you mean by "facet is selected" here?

Reply:
I took this copy from other monitors like RUM or Logs. I changed it by using `dimensions` and `measures`. Let me know if now it's clearer.